dotnet/src/Connectors/Connectors.MistralAI.UnitTests/Services/MistralAITextEmbeddingGenerationServiceTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Connectors.MistralAI;
+using Xunit;
+
+namespace SemanticKernel.Connectors.MistralAI.UnitTests.Services;
+
+/// <summary>
+/// Unit tests for <see cref="MistralAITextEmbeddingGenerationService"/>.
+/// </summary>
+public sealed class MistralAITextEmbeddingGenerationServiceTests : MistralTestBase
+{
+    [Fact]
+    public async Task ValidateGenerateEmbeddingsAsync()
+    {
+        // Arrange
+        var content = this.GetTestData("embeddings_response.json");
+        this.DelegatingHandler = new AssertingDelegatingHandler("https://api.mistral.ai/v1/embeddings", content);
+        this.HttpClient = new HttpClient(this.DelegatingHandler, false);
+        var service = new MistralAITextEmbeddingGenerationService("mistral-small-latest", "key", httpClient: this.HttpClient);
+
+        // Act
+        List<string> data = new() { "Hello", "world" };
```

Comment:
With 8.0 we can play with this new listing.

```suggestion
        List<string> data = ["Hello", "world"];
```